lesson3/goodsList.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+class GoodsList {
+  goods = [];
+
+  render = (list) => {
+    list.forEach((elem) => {
+      const goodItem = new GoodsItem(
+        elem.id_product,
+        elem.product_name,
+        elem.price
+      );
+      let goodsListHTML = goodItem.render();
+      document
+        .querySelector(".wrapper")
+        .insertAdjacentHTML("beforeend", goodsListHTML);
+    });
+  };
+
+  //   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  //   НЕ ПОНИМАЮ ПОЧЕМУ КЛИК ДВА РАЗА ?!?!
+  //   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  addListener() {
+    const buttons = document.querySelectorAll(".product_buyBtn");
+    buttons.forEach((elem) => {
+      addEventListener("click", function (event) {
```

Comment:
здесь вы пропустили указать, к чему вы добавляете eventListener, поэтому он добавился к глобальному window и срабатывал на все клики
```suggestion
      elem.addEventListener("click", function (event) {
```